pkg/controller/secret/secret_controller.go

```diff
@@ -16,6 +16,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/go-logr/logr"
```

Comment:
This doesn't look right

Reply:
What specifically doesn't look right? The same package is used in other operators we maintain, ie- [here](https://github.com/openshift/managed-velero-operator/blob/master/pkg/controller/velero/velero.go#L22).

Certainly open to removing if there's a way to implement these changes without the additional dependency though!